stable/mongodb/templates/statefulset-arbiter-rs.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
     {{- with .Values.labels }}
+{{ toYaml . | indent 4 }}
+    {{- end }}
+    {{- with .Values.statefulsetLabels }}
```

Comment:
Why do I require this "statefulsetLabels" parameter when the parameter "labels" is already providing the same feature?